docs/application/dotnet/guides/nui/window.md

```diff
@@ -313,6 +313,33 @@ child.Unparent();
 > If parent's window stack is changed by calling `Raise()` or `Lower()`, the child window is located on top of the parent again.
 >
 
+### Window Type
+
+NUI Window has serveral window types.
+The detail descrition is written in NUI Constants. See [WindowType] (application/dotnet/api/TizenFX/latest/api/Tizen.NUI.WindowType.html).
```

Comment:
please remove this line

Reply:
Line 319 is already included in 318.
So kindly remove LINE 319
The detail descrition is written in NUI Constants. See [WindowType] (application/dotnet/api/TizenFX/latest/api/Tizen.NUI.WindowType.html).

Reply:
Ah, Sorry. I removed line 319 and update patch.
Thank you for kindly review.